### PR TITLE
Update the README, introduce C4.1

### PR DESCRIPTION
**Pull request by rgbkrk**
This lays out our contribution model, background on our current stack, the design (in terms of models, UIs, software architecture), as well as our scope and goals.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,19 +42,32 @@ Requires node 5.x and npm 3.
...
+* [ZeroMQ](http://zguide.zeromq.org/page:all)
 
-The primary goal of composition is to create a fully standalone desktop application, written with Electron.
+These are our suggested tutorials and background to help you get up to speed:
 
-* [ ] Bundled kernel with scientific Python packages (Anaconda FTW!)
-* [ ] Observes kernelspecs available
-* [ ] One click installer for OS X and Windows.
+* [The introduction to Reactive Programming you've been missing](https://gist.github.com/staltz/868e7e9bc2a7b8c1f754)
+* [Full Stack Redux Tutorial](http://teropa.info/blog/2015/09/10/full-stack-redux-tutorial.html)
```
Comment (jdfreder) [inline README.md:63]: Glad you put these links here, it will help everyone share the same mindset.

Comment (rgbkrk) [inline README.md:63]: Yeah, I'm also expecting us to evolve these over time to better reflect the body of knowledge we're building together.

Comment (rgbkrk) [inline README.md:63]: On a similar note, @odewahn has stated an interest in developing a tutorial on the Jupyter message spec via enchannel and ick.

File: README.md [hunk 2]
```diff
@@ -21,19 +42,32 @@ Requires node 5.x and npm 3.
...
+programming is the biggest help.
 
-There are going to be node packages that should get maintained and worked on independent of composition itself.
+If you'd like to experiment with the core Jupyter messaging and Observables
+without working with React, check out [ick](https://github.com/nteract/ick),
+an interactive console.
 
-* [ ] Prebuilt versions of `zmq` for use in node/Electron/Atom (for everyone :tada:!)
-* [X] Port Transformime (Jupyter Mimetype bundles -> HTML Elements) over to pure `React` elements
+Post in Slack if you need help with these or have questions. Several people
```
Comment (jdfreder) [inline README.md]: Might as well turn this into a link:

`(Slack)[http://slack.nteract.in]`

Comment (rgbkrk) [inline README.md]: Good note!

**Pull request merged**